syn/python/translate_timing_csv.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+
+from flow_utils import *
+
+
+def main():
+    arg_parser = argparse.ArgumentParser(
+        description=
+        """Translates CSV timing report to have human readable start and end points
+given yosys generated name translation file (generated with build_translated_names.py"""
```

Comment:
as above: manually split into two lines, and re-run yapf should give better output.